packages/client/src/subscribe/ordering/OrderedMessageChain.ts

```diff
@@ -107,19 +107,23 @@ export class OrderedMessageChain {
         }
     }
 
-    resolveMessages(): void {
+    resolveMessages(to: MessageRef | undefined, gapCheckEnabled: boolean = true): void {
```

Comment:
I would not use a default boolean value here for the 2nd parameter.

```suggestion
    resolveMessages(to: MessageRef | undefined, gapCheckEnabled: boolean): void {
```